etc/layout_viewer/css/bootstrap-treeview.min.css

```diff
@@ -0,0 +1 @@
+.list-group-item{cursor:pointer}span.indent{margin-left:10px;margin-right:10px}span.icon{margin-right:5px}
```

Comment:
Could you please add `*.min.css` as an excluded filename for the `mach grep` command in `python/servo/devenv_commands.py`? (`*.min.js` is already excluded.)